docs/DEVELOPMENT.md

```diff
@@ -35,6 +34,32 @@ npx tauri dev
 
 Automatic recompiling on save is enabled for both the Rust and the Typescript/Vue code.
 
+If you want to build, do all of the above (except tauri dev) and:
+
+```sh
+cd ../src-tauri
+```
+
+```sh
+npm run tauri build
+```
+
+This will build the executable and bundles, such as AppImage or .deb.
```

Comment:
Don't forget that on Windows it builds an `.msi`.

Reply:
"such as `AppImage`, `.deb` or `.msi`", I'll add it, thanks.